[2022-04-07 19:24:21,727] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: load_to_BigQuery.create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task scheduled__2017-01-01T00:00:00+00:00 [queued]>
[2022-04-07 19:24:21,885] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: load_to_BigQuery.create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task scheduled__2017-01-01T00:00:00+00:00 [queued]>
[2022-04-07 19:24:21,885] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-07 19:24:21,886] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-04-07 19:24:21,886] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-07 19:24:23,108] {taskinstance.py:1270} INFO - Executing <Task(BigQueryInsertJobOperator): create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task> on 2017-01-01 00:00:00+00:00
[2022-04-07 19:24:23,148] {standard_task_runner.py:52} INFO - Started process 3720 to run task
[2022-04-07 19:24:23,167] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'load_to_BigQuery', 'create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task', 'scheduled__2017-01-01T00:00:00+00:00', '--job-id', '12872', '--raw', '--subdir', 'DAGS_FOLDER/load_to_BigQuery_dag.py', '--cfg-path', '/tmp/tmp1em6pbgo', '--error-file', '/tmp/tmpssm2zwj7']
[2022-04-07 19:24:23,189] {standard_task_runner.py:80} INFO - Job 12872: Subtask create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task
[2022-04-07 19:24:23,570] {logging_mixin.py:109} INFO - Running <TaskInstance: load_to_BigQuery.create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task scheduled__2017-01-01T00:00:00+00:00 [running]> on host 2c54f2187010
[2022-04-07 19:24:24,162] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=load_to_BigQuery
AIRFLOW_CTX_TASK_ID=create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task
AIRFLOW_CTX_EXECUTION_DATE=2017-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2017-01-01T00:00:00+00:00
[2022-04-07 19:24:24,165] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-04-07 19:24:24,214] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-04-07 19:24:24,247] {bigquery.py:1554} INFO - Inserting job ***_load_to_BigQuery_create_external_posts_answers_bq_create_posts_answers_stack_overflow_data_partitioned_table_task_2017_01_01T00_00_00_00_00_03780628e008e382d49a1ff8ebbd808e
[2022-04-07 19:24:24,854] {taskinstance.py:1774} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 2153, in execute
    job = self._submit_job(hook, job_id)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 2119, in _submit_job
    retry=self.result_retry,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/common/hooks/base_google.py", line 439, in inner_wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 1560, in insert_job
    job.result(timeout=timeout, retry=retry)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/job/query.py", line 1498, in result
    do_get_result()
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/job/query.py", line 1488, in do_get_result
    super(QueryJob, self).result(retry=retry, timeout=timeout)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/job/base.py", line 728, in result
    return super(_AsyncJob, self).result(timeout=timeout, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/api_core/future/polling.py", line 135, in result
    raise self._exception
google.api_core.exceptions.BadRequest: 400 Could not cast literal "creation_date" to type TIMESTAMP at [1:109]

Location: us-central1
Job ID: airflow_load_to_BigQuery_create_external_posts_answers_bq_create_posts_answers_stack_overflow_data_partitioned_table_task_2017_01_01T00_00_00_00_00_03780628e008e382d49a1ff8ebbd808e

[2022-04-07 19:24:24,932] {taskinstance.py:1288} INFO - Marking task as FAILED. dag_id=load_to_BigQuery, task_id=create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task, execution_date=20170101T000000, start_date=20220407T192421, end_date=20220407T192424
[2022-04-07 19:24:24,965] {standard_task_runner.py:98} ERROR - Failed to execute job 12872 for task create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task (400 Could not cast literal "creation_date" to type TIMESTAMP at [1:109]

Location: us-central1
Job ID: airflow_load_to_BigQuery_create_external_posts_answers_bq_create_posts_answers_stack_overflow_data_partitioned_table_task_2017_01_01T00_00_00_00_00_03780628e008e382d49a1ff8ebbd808e
; 3720)
[2022-04-07 19:24:25,032] {local_task_job.py:154} INFO - Task exited with return code 1
[2022-04-07 19:24:25,089] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-04-07 22:17:42,599] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: load_to_BigQuery.create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task scheduled__2017-01-01T00:00:00+00:00 [queued]>
[2022-04-07 22:17:42,639] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: load_to_BigQuery.create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task scheduled__2017-01-01T00:00:00+00:00 [queued]>
[2022-04-07 22:17:42,639] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-07 22:17:42,639] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-04-07 22:17:42,640] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-07 22:17:43,045] {taskinstance.py:1270} INFO - Executing <Task(BigQueryInsertJobOperator): create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task> on 2017-01-01 00:00:00+00:00
[2022-04-07 22:17:43,083] {standard_task_runner.py:52} INFO - Started process 6036 to run task
[2022-04-07 22:17:43,114] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'load_to_BigQuery', 'create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task', 'scheduled__2017-01-01T00:00:00+00:00', '--job-id', '14129', '--raw', '--subdir', 'DAGS_FOLDER/load_to_BigQuery_dag.py', '--cfg-path', '/tmp/tmpfsxo9ivb', '--error-file', '/tmp/tmp2gsr7q3t']
[2022-04-07 22:17:43,115] {standard_task_runner.py:80} INFO - Job 14129: Subtask create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task
[2022-04-07 22:17:43,332] {logging_mixin.py:109} INFO - Running <TaskInstance: load_to_BigQuery.create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task scheduled__2017-01-01T00:00:00+00:00 [running]> on host 36107d30f64e
[2022-04-07 22:17:43,648] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=load_to_BigQuery
AIRFLOW_CTX_TASK_ID=create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task
AIRFLOW_CTX_EXECUTION_DATE=2017-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2017-01-01T00:00:00+00:00
[2022-04-07 22:17:43,651] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-04-07 22:17:43,672] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-04-07 22:17:43,695] {bigquery.py:1554} INFO - Inserting job ***_load_to_BigQuery_create_external_posts_answers_bq_create_posts_answers_stack_overflow_data_partitioned_table_task_2017_01_01T00_00_00_00_00_3aa29f2891537542ec2bec6e9f1f5133
[2022-04-07 22:17:47,255] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=load_to_BigQuery, task_id=create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task, execution_date=20170101T000000, start_date=20220407T221742, end_date=20220407T221747
[2022-04-07 22:17:47,368] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-07 22:17:47,480] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
